src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -267,6 +267,10 @@
       <EmscriptenEnvVars Include="EM_FROZEN_CACHE=True" />
       <EmscriptenEnvVars Include="DISABLE_LEGACY_JS_INTEROP=1" Condition="'$(WasmEnableLegacyJsInterop)' == 'false'" />
       <EmscriptenEnvVars Include="DISABLE_LEGACY_JS_INTEROP=0" Condition="'$(WasmEnableLegacyJsInterop)' != 'false'" />
+      <EmscriptenEnvVars Include="ENABLE_AOT_PROFILER=1" Condition="$(WasmProfilers.Contains('aot'))" />
```

Comment:
I think the 1/0 could be computed expressions, but I'm fine with not doing that yet.

Reply:
it's a bool, and I think that MSBuild doesn't have ternary operator. Any specific hint ?